packages/expo-modules-core/android/src/main/java/expo/modules/kotlin/records/Record.kt

```diff
@@ -40,3 +45,33 @@ fun Record.toJSMap(): ReadableMap {
 
   return writableMap
 }
+
+// TODO (barthap): Move these to the right place
```

Comment:
Need this, because otherwise I couldn't use Records with Enum fields in tests.

This impl is not perfect, I just wrote it to make tests passing